src/components/custom/_agreement-label.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { LinkText, Checkbox } from 'components/elements'
+import { Localize } from 'components/localization'
+import device from 'themes/device.js'
+
+const CheckboxSpan = styled.span`
+    font-size: var(--text-size-xs);
+
+    @media ${device.tabletL} {
+        font-size: 1.75rem;
+    }
+`
+const StyledLinkText = styled(LinkText)`
+    font-size: ${(props) => props.size || '14px'};
+`
```

Comment:
```suggestion
```
This can be omitted as you can pass the `size` prop directly to the `LinkText` component